canary/generator/transformators/transformers_linear.py

```diff
@@ -0,0 +1,162 @@
+from abc import ABC
+from copy import deepcopy
+
+import numpy as np
+
+from .base import Transformer
+
+
+class TransformerLinear(Transformer, ABC):
+    def __init__(self):
+        super(TransformerLinear, self).__init__(['linear'])
+
+
+class AddMoveTransformer(TransformerLinear):
```

Comment:
I feel we should invest some time into adding docs for each transformation in this PR, rather than deferring it: we have a clearer picture now that will probably fade away in weeks :)